src/chapters/setup-intro-data-types/data-types.rst

```diff
@@ -193,24 +193,31 @@ be found on the
    Anytime you need to perform calculations with decimal values, consider using
    the ``double`` type instead of ``float``.
 
+Non-primitive Types
+--------------------
+
 Primitive data types are *immutable* and can be combined to build larger data
-structures, like arrays and objects. One example is combining multiple ``char``
-to form a ``String`` data type.
+structures. One example is forming the ``String`` "LaunchCode" from multiple
+``char`` characters ('L', 'a', 'u', etc.).
 
-.. note::
+``String`` is a non-primitive data type, also called an *object type*. As we
+saw in the ``String`` table above, object types have methods which we can call
+using dot notation. Primitive data types do not have methods.
 
-   Primitive data types in Java are identified as starting with a lower case letter, while
-   object data types in Java begin with a capital letter.
+.. admonition:: Note
 
-Autoboxing
-^^^^^^^^^^^
+   Primitive data types in Java begin with a lower case letter, while object
+   data types in Java begin with a capital letter.
 
-As we saw with the String data type, *object types* have methods which we can
-call using dot notation. Primitive data types do not have methods.
+Later in this chapter, we will explore the array and class object types.
```

Comment:
I'd capitalize Array and Class for clarity

Reply:
Fixed.  Thanks.